plugin.go

```diff
@@ -0,0 +1,111 @@
+package containerd
+
+import (
+	"fmt"
+	"path/filepath"
+	"plugin"
+	"runtime"
+	"sync"
+
+	"github.com/docker/containerd/content"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type PluginType int
+
+const (
+	RuntimePlugin PluginType = iota + 1
```

Comment:
I assume that these are just the plugin types that you've stubbed out for now and not intended to be an exhaustive list?